src/webview/components/memory-table.tsx

```diff
@@ -14,16 +14,18 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+// Primereact works with null values
+/* eslint-disable no-null/no-null */
```

Comment:
Some thoughts here:
 - There's only one null value in the file, set in two places.
 - In the only place where it's accessed (l. 221), it's typed as not `null`. If it could be `null` there, then that cast is wrong and should be removed, and the receiver of the (potentially) `null` value should be updated to handle that. If it really can't be `null`, then we never access it when it's `null`, and there would be no reason to make it `null` rather than `undefined`.
 - I really don't care much about `null` vs. `undefined`, so I'd be happy removing this lint rule if we actually do need `null` values sometimes. What do you think, @thegecko?

Reply:
I personally prefer to use undefined where possible and avoid null, but I hold that view lightly and wouldn't lose sleep over nulls being used ;)

Reply:
I also prefer `undefined`. However, primereact uses `null` instead of `undefined` for the `selection` variable if we use `cellselection`. If we use `rowselection` (which I subjectively find more appropriate) then we will not require it anymore.